COMPILING.md

```diff
@@ -69,7 +69,7 @@ that comes with vcpkg. After that, you can run something similar to this:
 ```powershell
 mkdir build
 cd build
-cmake.exe .. -G'Visual Studio 16 2019' -DCMAKE_TOOLCHAIN_FILE="<location of vcpkg>\vcpkg\scripts\buildsystems\vcpkg.cmake" -DVCPKG_TARGET_TRIPLET="x64-windows-static"
+cmake.exe .. -G"Visual Studio 16 2019" -DCMAKE_TOOLCHAIN_FILE="<location of vcpkg>\vcpkg\scripts\buildsystems\vcpkg.cmake" -DVCPKG_TARGET_TRIPLET="x64-windows-static"
```

Comment:
Assuming you've set the upstream properly as described in [CONTRIBUTING.md](https://github.com/OpenTTD/OpenTTD/blob/master/CONTRIBUTING.md#pull-requests) and are using the command line (as opposed to GitHub Desktop, which doesn't really have full functionality), you should be able to rebase this fork on OpenTTD:master with the commands:
`git fetch upstream`
`git rebase upstream/master`

Then you can do `git rebase -i HEAD~4` to start an [interactive rebase](https://docs.github.com/en/get-started/using-git/about-git-rebase), which allows you to drop unwanted commits, squash commits together, and reword them to match [OpenTTD commit message format.](https://wiki.openttd.org/en/Development/Coding%20style#commit-message)

To push your changed commits to this PR, you'll need to force-push your commits with `git push -f origin HEAD`.

Also, assuming you're new to GitHub, I quite like the built-in Git GUI tool for building commits, particularly when you're adding things in multiple commits. It's also good for checking trailing whitespace, which is failing the commit checker.. You can start it with `git gui`. 😃

Reply:
OK this is a bit complex for me, I'll try to get to this when I have more time. Thanks for the detailed instructions.

Reply:
I ran into a lot of conflicts - I think the simplest thing is for me to delete this PR, delete my branch, make a new branch, redo my changes in one commit manually, and make a new PR. Is that OK?

Reply:
Sorry for all the hassle; new PR is here: https://github.com/OpenTTD/OpenTTD/pull/9770